opal/util/show_help.c

```diff
@@ -88,6 +104,50 @@ static void opal_show_help_finalize(void)
     }
 }
 
+static void opal_show_help_cbfunc(pmix_status_t status, void *cbdata)
+{
+    opal_log_info_t *info = (opal_log_info_t *) cbdata;
+    if(PMIX_SUCCESS != status && PMIX_OPERATION_SUCCEEDED != status) {
+        opal_output(output_stream, "%s", info->msg);
+    }
+    PMIX_INFO_DESTRUCT(info -> info);
+    if(info -> dirs) {
+        PMIX_INFO_DESTRUCT(info -> dirs);
+    }
+    free(info -> msg);
+    free(info);
+}
+
+static void local_delivery(const char *file, const char *topic, char *msg) {
+    pmix_info_t *info, *dirs;
+    int ninfo = 0, ndirs = 0;
+    PMIX_INFO_CREATE(info, 1);
+    PMIX_INFO_LOAD(&info[ninfo++], PMIX_LOG_STDERR, msg, PMIX_STRING);
+
+    opal_log_info_t *cbdata = calloc(1, sizeof(opal_log_info_t));
+    if(opal_help_want_aggregate) {
+        PMIX_INFO_CREATE(dirs, 3);
+        PMIX_INFO_LOAD(&dirs[ndirs++], PMIX_LOG_AGG, &opal_help_want_aggregate, PMIX_BOOL);
+        PMIX_INFO_LOAD(&dirs[ndirs++], PMIX_LOG_KEY, file, PMIX_STRING);
+        PMIX_INFO_LOAD(&dirs[ndirs++], PMIX_LOG_VAL, topic, PMIX_STRING);
+        cbdata -> dirs = dirs;
```

Comment:
It's a little weird to put spaces on both sides of `->` -- I don't think we do that elsewhere in OMPI, do we?